packages/datetime2/src/components/date-input3/dateInput3.tsx

```diff
@@ -568,6 +568,9 @@ export const DateInput3: React.FC<DateInput3Props> = React.memo(function _DateIn
     );
 });
 DateInput3.displayName = `${DISPLAYNAME_PREFIX}.DateInput3`;
+
+// TODO: Removing `defaultProps` here breaks tests. Investigate why.
+// eslint-disable-next-line deprecation/deprecation
```

Comment:
That's fun. 😂

Thanks for leaving a comment so others have context on this in the future.

Reply:
Yeah.. 🥲

I am working on updating/refactoring a lot of the datetime2 tests currently, so will hopefully uncover the cause of this during the process.